index.html

```diff
@@ -590,20 +590,72 @@ <h4>serializeBaseProofValue</h4>
           <p>
 The following algorithm serializes the base proof value, including the
 BBS signature, HMAC key, and mandatory pointers.
-The required inputs are a base signature |bbsSignature|,
-an HMAC key |hmacKey|, and an array of
-|mandatoryPointers|.
+The required inputs are a base signature |bbsSignature|, |bbsHeader|,
+|publicKey|, an HMAC key |hmacKey|, an array of
+|mandatoryPointers|, |featureOption|, and depending on
+the |featureOption| value the |pid|, and |signer_blind| values.
 A single <em>base proof</em> string value is produced as output.
           </p>
 
           <ol class="algorithm">
             <li>
-Initialize a byte array, `proofValue`, that starts with the BBS base proof
-header bytes `0xd9`, `0x5d`, and `0x02`.
+Depending upon the value of the |featureOption|, set up the |proofValue| as
+follows.
             </li>
             <li>
+If |featureOption| equals `"baseline"`:
```

Comment:
I think we might want to shorten this to just `feature` (or another single word name like "mode" or something that we come to consensus on), but I wouldn't hold up this PR for it. I think this PR is an approvement and am in support.

Reply:
Hi @dlongley, agree. Picking good names is hard. Currently have a lot of names with *option* in it. Simpler the better.